media/js/firefox/election.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+    var quantum = 57;
+    var htmlClassName = document.documentElement.className;
+
+    if(client.isFirefoxDesktop === false || client.FirefoxVersion < quantum) {
+        // case-not-firefox-desktop-current
+        document.getElementById('case-firefox-desktop-current').classList.add('hidden');
+        document.getElementById('case-not-firefox-desktop').classList.remove('hidden');
+        if (htmlClassName.indexOf('oldwin') > -1 || htmlClassName.indexOf('oldmac') > -1  || htmlClassName.indexOf('arm') > -1 ) {
```

Comment:
As near as I can tell, this is returning true in both Chrome and Firefox on Android, but I can't understand why... this seems sound to me.

Reply:
Logic is sound, but Android reports as arm. Who knew?